ivadomed/main.py

```diff
@@ -177,9 +177,10 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
         imed_utils.display_selected_transfoms(transformation_dict, dataset_type=["testing"])
 
     # Check if multiple raters
-    if any([class_suffix for class_suffix in loader_params["target_suffix"]]):
+    if any([isinstance(class_suffix, list) for class_suffix in loader_params["target_suffix"]]):
```

Comment:
With the first condition, the result was always true (`any` on a string gives true). I changed the condition to verify if the element of target_list are lists or strings indicating if there are multiple raters or not.